packages/spectacular/src/lib/bootstrap-test-application/util-dom/get-content.ts

```diff
@@ -0,0 +1,9 @@
+import { isTemplateElement } from './is-template-element';
+
+export function getContent(node: Node): Node {
+  if (isTemplateElement(node)) {
+    return node.content;
+  } else {
+    return node;
```

Comment:
return isTemplateElement(node)) ? node.content : node;

Reply:
You can use triple backticks followed by _suggestion_ (```suggestion) to make code suggestions for the PR.